server/services/personalPageService.ts

```diff
@@ -552,4 +557,17 @@ export default class PersonalPageService {
 
     return response
   }
+
+  async updateNationality(clientToken: string, user: PrisonUser, prisonerNumber: string, nationality: string) {
+    const personIntegrationApiClient = this.personIntegrationApiClientBuilder(clientToken)
+    const response = personIntegrationApiClient.updateNationality(prisonerNumber, nationality)
```

Comment:
I can see that we've missed it in a few other places too but we could do to do an `await` on the api call so that the metrics only get registered if the update is successful.